applications/tari_app_grpc/proto/wallet.proto

```diff
@@ -69,6 +69,11 @@ message PaymentRecipient {
     uint64 amount = 2;
     uint64 fee_per_gram = 3;
     string message = 4;
+    enum PaymentType {
+        STANDARD_MIMBLEWIMBLE = 0;
```

Comment:
Oh nice - seems like prost doesn't require enums to be named `PAYMENT_TYPE_ONE_SIDED` anymore

Reply:
Oh didn't realize that would be a possible issue but ja seemed to generate just fine.